ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -4445,6 +4454,8 @@ CHDX2SC:
   - compressor_speed_percentage_sensor
   - cooling_percentage_command
   - cooling_request_count
+  - compressor_speed_percentage_sensor_1
```

Comment:
A separate abstract type should be created for multiple stage compressors that have multiple speed percentage sensors. I.e. use this to capture the temperature control in conjunction with another to capture speed control.